_targets/ia32/generic/nvm.yaml

```diff
@@ -1,5 +1,5 @@
 hd0:
-  size: 0x4000000 # 64 MB
+  size: 0x8000000 # 128 MB
```

Comment:
@Darchiv can we increase the size?

Reply:
Seems fine.

@adamgreloch did not encounter any issues as far as I knoiw